tools/cli/defs.go

```diff
@@ -26,11 +26,17 @@ import (
 
 	"github.com/fatih/color"
 	"github.com/olekukonko/tablewriter"
-	s "go.uber.org/cadence/.gen/go/shared"
+
+	"github.com/uber/cadence/common/types"
 )
 
 const (
-	localHostPort = "127.0.0.1:7933"
+	localHost    = "127.0.0.1"
+	tchannelPort = localHost + ":7933"
+	grpcPort     = localHost + ":7833"
+
+	grpcChannel   = "grpc"
```

Comment:
Channel is specific to TChannel. Maybe just call it grpcTransport, tchannelTransport intead?